runtime/compiler/il/J9Node.cpp

```diff
@@ -2212,6 +2212,23 @@ J9::Node::setDontInlinePutOrderedCall()
    }
 
 bool
+J9::Node::isUnsafeCopyMemoryIntrinsic()
```

Comment:
I found that having a query on the call node is the most convenient place as we can have all the information to check whether call node is JNI call and it is one of the recognized copyMemory method.